src/SFA.DAS.RoatpOversight.Web/Services/PdfValidatorService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace SFA.DAS.RoatpOversight.Web.Services
+{
+    public class PdfValidatorService : IPdfValidatorService
+    {
+        public async Task<bool> IsPdf(IFormFile file)
+        {
+            var pdfHeader = new byte[] { 0x25, 0x50, 0x44, 0x46 };
```

Comment:
Not checking IFormFile. ContentType for PDF MIME type (application/pdf) . Was that intentional?

Reply:
This code is taken from FileContentIsValidForPdfFile in AdminServices, and checking content type isn't foolproof. However, I'll add the check in.